settings.gradle

```diff
@@ -102,5 +105,4 @@ includeBuild 'content/modules/uiconstraints/examples/uiconstraints-ex1'
 
 includeBuild 'content/modules/webdav/examples/webdav-ex1'
 
-includeBuild 'content/modules/pivot-table/examples/pivottable-ex11'
```

Comment:
this seemed to be an error - there was no `pivottable-ex11` example, which lead to an error while building. I now removed the line.